CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 
 * fix(exporter-zipkin): rounding duration to the nearest int to be compliant with zipkin protocol [#4064](https://github.com/open-telemetry/opentelemetry-js/pull/4064) @n0cloud
 * fix(sdk-metrics): metric names should be case-insensitive
+* fix(sdk-metrics): do not report empty scopes and metrics [#4135](https://github.com/open-telemetry/opentelemetry-js/pull/4135) @pichlermarc
```

Comment:
Should this be under "Unreleased"?

Reply:
Ah yes, I must've added the entry before the release. good catch :slightly_smiling_face: [40a9ced](https://github.com/open-telemetry/opentelemetry-js/pull/4135/commits/40a9cedb6039ac9860d632dbcd304d3ecdac9808)